src/device.rs

```diff
@@ -2410,7 +2410,7 @@ impl TaskRuntime for Runtime {
             },
 
             _ = self.polling_interval.tick() => {
-                telio_log_debug!("WG consolidation triggered by tick event");
+                telio_log_debug!("WG consolidation triggered by tick event, total logs dropped: {}", logs_dropped_until_now());
```

Comment:
nit: do we expect it in normal circumstances? Maybe `telio_log_warn!` is more appropriate

Reply:
Not sure I understand, but this is not the delta since last call, but a total number of dropped logs (strictly monotonic) so the log should stay debug.

Reply:
I mean if the count of skipped events is greater than 0 then we could warn! instead of debug!, however this would require keeping the previous skip count to detect this.

basically my point is to make it more visible as debug messages might get lost in a sea of other logs :)

Reply:
You're right, with buffer size at 1024, it should be fine to issue a warning about new dropped logs. I've added it and some tests to actually prove the dropping is working as expected :)